src/index.js

```diff
@@ -1,16 +1,103 @@
 import './style.css';
-import { UI, Todo } from '../modules/index.js';
+import Todo from '../modules/todo.js';
+import Store from '../modules/store.js';
+import toggle from '../modules/check.js';
 
-// Events
+// UI class: Handle UI tasks
+class UI {
+  static showTasks() {
+    const tasks = Store.getTasks();
 
-document.addEventListener('DOMContentLoaded', UI.displayTodo);
+    tasks.forEach((task) => UI.addBookToList(task));
+  }
 
-// document.querySelector('.add-to-list').addEventListener('submit', (e) => {
-//    const todoText = document.querySelector('.todo-input').value;
-// });
+  static addBookToList(task) {
+    const list = document.querySelector('.js-list');
 
-// eslint-disable-next-line no-undef, no-unused-vars
-const book = new Todo(desc, completed, id);
+    // const div = document.createElement('div');
+    // div.classList.add('task-item');
 
-// eslint-disable-next-line no-undef
-UI.addBookToList(todo);
+    list.innerHTML += `
+      <li class="todo-li">
+      <button class="checkbox" ><i class="fa-solid fa-check ${
+  task.completed ? 'active' : ''
+}"></i></button>
+        <p class="todo-p-1" id='para' contenteditable='true'>${task.item}</p>
+        <button class='delete'>X</button> 
+      </li>`;
+  }
+
+  // remove function
+  static removeTask(element) {
+    if (element.classList.contains('delete')) {
+      element.parentElement.remove();
+    }
+  }
+
+  static clearfields() {
+    document.querySelector('#dataInput').value = '';
+  }
+}
+
+// Event: Show Items
+document.addEventListener('DOMContentLoaded', () => {
+  UI.showTasks();
+  document.querySelectorAll('#para').forEach((paragraph, index) => {
+    paragraph.addEventListener('input', () => {
+      // const todo = new Todo(paragraph.textContent);
+      const Storage1 = JSON.parse(localStorage.getItem('tasks')) || [];
+      Storage1[index].item = paragraph.textContent;
+      localStorage.setItem('tasks', JSON.stringify(Storage1));
+      // console.log(Storage1[index].item);
+      // console.log(paragraph.textContent);
+    });
+  });
+  document.querySelectorAll('.delete').forEach((deleteButton) => {
+    deleteButton.addEventListener('click', (e) => {
+      // remove task from UI
+      // console.log('This is working');
+      Store.removeTask(e.target.previousElementSibling.textContent);
+      UI.removeTask(e.target);
+
+      // Remove task from store
+    });
+  });
+});
+
+// Add a task
+document.querySelector('.add-to-list').addEventListener('submit', (e) => {
+  // prevent the actual submit
+  e.preventDefault();
+
+  // get form values
+  const item = document.querySelector('#dataInput').value;
```

Comment:
- [ ] Kindly make sure that while adding a new task to the todo list it should save to the localstorage and also it should appear on page. but now when I add new todo list task nothing happens even after cloning to my local machine and installing all dependencies.
https://www.loom.com/share/10ffc3d11f974a4393fb86a237c48e35

Reply:
@sajid-munawar Bro everything starts working after i write npm start on terminal but how can i make sure that everything works without needing npm start

Reply:
@sajid-munawar Bro Kindly check in my Loom link everything is working and styled but i have to npm start on terminal to get everything to happen how can i make sure that i dont have to use npm ???? please guide me thankyou
https://www.loom.com/share/b819ea49b0b146029dd82605eb86cad4

Reply:
Fixed.

Reply:
thats cool everything is working fine.